code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -1148,5 +1150,5 @@ to_chat will check for valid clients itself already so no need to double check f
 	if(D?.siloless_hive_timer)
 		return
 
-	xeno_message("<span class='xenoannounce'>A sudden tremor ripples through the hive... the last silo was destroyed! The hive will collapse if nothing is done</span>", 3, TRUE)
+	xeno_message("<span class='xenoannounce'>We don't have any silos! The hive will collapse if nothing is done</span>", 3, TRUE)
```

Comment:
Why?

Reply:
If you lose all silos at 12 : 50 for exemple, you won't get a message before 13 : 00. 
Same can be said if you never put any silo before 13 : 00 , so this message is more relevant

Reply:
Why did you change the message + not documented it in the PR?

Reply:
Cause i thought this was absolutly not important at all and still do

Reply:
'k